src/CtrlGroup.c

```diff
@@ -443,32 +471,32 @@ BOOL Ros_CtrlGroup_GetTorque(CtrlGroup* ctrlGroup, double torqueValues[MAX_PULSE
     return TRUE;
 }
 
-//-------------------------------------------------------------------
-// Retrieves the temperatures of all encoders in the specified group (integer, degrees C)
-//-------------------------------------------------------------------
-BOOL Ros_CtrlGroup_GetEncoderTemperature(CtrlGroup const* const ctrlGroup, long encoderTemp[MAX_PULSE_AXES])
-{
-    MP_CTRL_GRP_SEND_DATA sData;
-    MP_ENCODER_TEMP_RSP_DATA rData;
-    LONG status = 0;
-
-    bzero(&sData, sizeof(sData));
-    bzero(&rData, sizeof(rData));
-
-    sData.sCtrlGrp = ctrlGroup->groupId;
-
-    // get encoder temperatures
-    status = mpGetEncoderTemp(&sData, &rData);
-    if (0 != status)
-    {
-        Ros_Debug_BroadcastMsg("Failed to get encoder temperature: %ld", status);
-        return FALSE;
-    }
-
-    memcpy(encoderTemp, rData.lTemp, MAX_PULSE_AXES * sizeof(long));
-
-    return TRUE;
-}
+////-------------------------------------------------------------------
+//// Retrieves the temperatures of all encoders in the specified group (integer, degrees C)
+////-------------------------------------------------------------------
+//BOOL Ros_CtrlGroup_GetEncoderTemperature(CtrlGroup const* const ctrlGroup, long encoderTemp[MAX_PULSE_AXES])
+//{
+//    MP_CTRL_GRP_SEND_DATA sData;
+//    MP_ENCODER_TEMP_RSP_DATA rData;
+//    LONG status = 0;
+//
+//    bzero(&sData, sizeof(sData));
+//    bzero(&rData, sizeof(rData));
+//
+//    sData.sCtrlGrp = ctrlGroup->groupId;
+//
+//    // get encoder temperatures
+//    status = mpGetEncoderTemp(&sData, &rData);
+//    if (0 != status)
+//    {
+//        Ros_Debug_BroadcastMsg("Failed to get encoder temperature: %ld", status);
+//        return FALSE;
+//    }
+//
+//    memcpy(encoderTemp, rData.lTemp, MAX_PULSE_AXES * sizeof(long));
+//
+//    return TRUE;
+//}
```

Comment:
Seem https://github.com/Yaskawa-Global/motoros2/issues/18#issuecomment-1585503628. I'm not sure this should be needed.

Reply:
Would it be OK if I revert 46cfdcda4905333fc53590fac5ab6e83b97a1d85? Just to keep the history clean in this PR?

Reply:
Yes.
I did [0b15446](https://github.com/Yaskawa-Global/motoros2/pull/49/commits/0b154464e408c3378fb06a4ace098257a51d2a81), but I'm not sure that's what you intended.

Reply:
I was thinking of rebasing and dropping it. I'll probably do that later.

Reply:
Done.